bodyprocessors/multipart.go

```diff
@@ -50,18 +51,29 @@ func (mbp *multipartBodyProcessor) ProcessRequest(reader io.Reader, collections
 		// if is a file
 		filename := originFileName(p)
 		if filename != "" {
-			temp, err := os.CreateTemp(storagePath, "crzmp*")
-			if err != nil {
-				return err
-			}
-			sz, err := io.Copy(temp, p)
-			if err != nil {
-				return err
+			var size int64
+			if !environment.IsTinyGo {
```

Comment:
nit: can you do first the negative to save one negation?

Reply:
Sorry JC I missed this comment - I can followup with the change if you feel strongly about it. I've generally wanted to keep "normal code" first, and TinyGo code later in order where possible